modules/system/lang/en/lang.php

```diff
@@ -26,7 +26,7 @@
         'se' => 'Swedish',
         'sk' => 'Slovak (Slovakia)',
         'tr' => 'Turkish',
-        'zh-cn' => 'Chinese (China)',
+        'zh-cn' => '简体中文',
```

Comment:
I think use native language will more friendly, maybe user does not know little English?